airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -56,15 +56,15 @@ class KubernetesHook(BaseHook):
         :doc:`/connections/kubernetes`
 
     :param conn_id: The :ref:`kubernetes connection <howto/connection:kubernetes>`
-        to Kubernetes cluster.
+        to Kubernetes cluster
     :param client_configuration: Optional dictionary of client configuration params.
-        Passed on to kubernetes client.
+        Passed on to kubernetes client
     :param cluster_context: Optionally specify a context to use (e.g. if you have multiple
-        in your kubeconfig.
-    :param config_file: Path to kubeconfig file.
-    :param in_cluster: Set to ``True`` if running from within a kubernetes cluster.
-    :param disable_verify_ssl: Set to ``True`` if SSL verification should be disabled.
-    :param disable_tcp_keepalive: Set to ``True`` if you want to disable keepalive logic.
+        in your kubeconfig
+    :param config_file: Path to kubeconfig file
+    :param in_cluster: Set to ``True`` if running from within a kubernetes cluster
+    :param disable_verify_ssl: Set to ``True`` if SSL verification should be disabled
+    :param disable_tcp_keepalive: Set to ``True`` if you want to disable keepalive logic
     """
```

Comment:
I appretiate the punctiliousness, however would you please revert these lines?  
Period at end of line in param docstring is not an established convention for airflow.  So it's not really worth the obfuscation of the git history.  Adding these kinds of unrelated changes to PRs is a distraction for reviewers.  I don't mind it when it's something uncontroversial / obviously positive, like fixing a typo.  But here a developer could go either way and no one would care, so we shouldn't change it for no reason.

Same for the capitalization you've introduced in comments below.

Reply:
I have changed these due to pycharm weak warnings. And the pros of these just not to stop an eye on these week warnings

Reply:
Yep. We have the rule, that if we want to follow certain standards, we enforce it with pre-commit so that it can be kept coinsistently across the codebase- if there is no rule for that, modifying code because "someone's" warnings are showing up is not a consistent approach and suffers the "works-for-me" syndrome. 

We should either turn such things in a rule (which I think we should not in this case) or refrain from changing the code to satisfy the "individual" rule. PyCharm has a way to disable such rules individually - you can store it in your project after checking out. Since we do not "enforce" common configuration for PyCharm/Intellij etc. we do not want to share IDE configuration - instead we opted for .gitignoring known tool configurations so whatever you change in your Pycharm's project config will not be committed to the repo - feel free to change it.

Reply:
OK, got it